### PR TITLE
Fix for Url generation 

### PR DESCRIPTION
**Pull request by xeon-zolt**
#30

### CODE REVIEW

**Comment by meets2tarun** [comment]
PR also solving https://github.com/fossasia/meilix-generator/issues/35

File: templates/index.html [hunk 1]
```diff
@@ -23,7 +23,9 @@
         EVENT NAME
     </TD>
     <TD>
+        <input type="text"  pattern="[A-Za-z0-9\s]{1,20}" placeholder="Enter the name of Event" name="TRAVIS_TAG">
```
Comment (meets2tarun) [inline templates/index.html]: The reason for placing this twice? @xeon-zolt

File: urltest [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```
Comment (meets2tarun) [inline urltest]: Is URL test working for now? @xeon-zolt

Comment (xeon-zolt) [inline urltest]: the script is working
i am still working on updating it on the page

File: templates/index.html (hunk 1 above)
Comment (xeon-zolt) [inline templates/index.html]: fixed
it was created by merge conflict

File: urltest (hunk 2 above)
Comment (meets2tarun) [inline urltest]: Shouldn't we remove this until we have a complete working URL test file which will solve #27? 
A different PR can be made for that.

**Pull request merged**